lib/discourse_theme/cli.rb

```diff
@@ -167,6 +183,94 @@ def run(args)
         else
           UI.info "Manage: #{client.root}/admin/customize/themes/#{theme_id}"
         end
+      elsif command == "rspec"
+        index = dir.index("/spec")
+        spec_path = "/spec"
+
+        if index
+          spec_path = dir[index..-1]
+          dir = dir[0..index - 1]
+        end
+
+        raise DiscourseTheme::ThemeError.new "'#{dir} does not exist" unless Dir.exist?(dir)
+
+        # Checks if the container is running
+        container_name = DISCOURSE_TEST_DOCKER_CONTAINER_NAME
+        is_running = false
+        container_exists = false
+
+        if !(
+             output =
+               execute(command: "docker ps -a --format '{{json .}}' | grep #{container_name}")
```

Comment:
If the container doesn't exist, the `grep` returns a non-zero exit status and I get

```
✘ Error occured while running: `docker ps -a --format '{{json .}}' | grep discourse_theme_test`:
```
I can work past it by adding `exit_on_error: false`, but then that'll swallow up any legit docker errors. 

Maybe we just run the docker command, and then parse the JSON in ruby?

Reply:
This might be OSX specific because on linux I get an exit status of `0` even if the container doesn't exist. 

> Maybe we just run the docker command, and then parse the JSON in ruby?

Yup I think we'll have to go this way to support multiple platform.

Reply:
O `docker ps` has a `--filter` option so I went with that instead in https://github.com/discourse/discourse_theme/pull/28/commits/0d79a16d3dd167a30a267cd2d8340e41850e9e84.